scrapers/KinkBomb.yml

```
@@ -0,0 +1,48 @@
+name: "KinkBomb"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - kinkbomb.com/p
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="content-section"]
+      $metainfo: //div[@class="metainfo"]
+      $image: //div[@class="clip-preview-inner"]
+    scene:
+      Title: $details//h1/text()
+      Image:
+        selector: $image/img/@src|$image/video/@poster
+        postProcess:
+          - replace:
+              - regex: //
+                with: https://
+
+      Details:
+        selector: //div[@class="description-inner"]/p
+        concat: "\n\n"
+      Date:
+        selector: $metainfo/small[2]/text()
+        postProcess:
+          - parseDate: 01/02/2006
+      Performers:
+        Name:
+          selector: $details//h4[@class="studio-header-name"]/text()
+          postProcess:
+            - replace:
+                - regex: .com
+                  with: ""
+      Studio:
+        Name:
+          selector: $details//h4[@class="studio-header-name"]/text()
+          postProcess:
+            - replace:
+                - regex: .com
+                  with: ""
```

You are using the same thing for Studio/Performers, In this case a yaml anchor helps a lot and makes sure you won't update only one of selectors by mistake
Also the `.` is a special regex char so it needs to be escaped
```suggestion
      Performers: &studioAttr
        Name:
          selector: $details//h4[@class="studio-header-name"]/text()
          postProcess:
            - replace:
                - regex: \.com
                  with: ""
      Studio: *studioAttr
```